components/list/list-item-mixin.js

```diff
@@ -344,6 +362,20 @@ export const ListItemMixin = superclass => class extends LocalizeCoreElement(Lis
 		if (node) node.focus();
 	}
 
+	async highlight() {
```

Comment:
Do you want to short-circuit out of here if `this._highlight` or `this._highlighting` is already `true`? i.e. if it's already in the process of highlighting.